modules/vardictjava/main.nf

```diff
@@ -0,0 +1,50 @@
+process VARDICTJAVA {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::vardict-java=1.8.3" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/vardict-java:1.8.3--hdfd78af_0':
+        'quay.io/biocontainers/vardict-java:1.8.3--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(bam), path(bai)
+    path(bed)
+    tuple path(fasta), path(fasta_fai)
+
+    output:
+    tuple val(meta), path("*.vcf.gz"), emit: vcf
+    path "versions.yml"              , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def args2 = task.ext.args2 ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    def VERSION = '1.8.3'
```

Comment:
nf-core modules typically put this on first line of the whole file, outside the process scope [^1]

[^1]: https://nf-co.re/docs/contributing/modules#general point 5 HOMER example